testing/checkin/runtime/__init.js

```diff
@@ -16,6 +17,7 @@ globalThis.setInterval = timers.setInterval;
 globalThis.clearTimeout = timers.clearTimeout;
 globalThis.clearInterval = timers.clearInterval;
 globalThis.Worker = worker.Worker;
+globalThis.throwInExt = throwInExt.throwInExt;
```

Comment:
Note that we already have a number of "throwing" functions -- you can probably just use `throwErrorWithContextSync` and `throwErrorWithContextAsync` from `checkin:error`

Reply:
I could reuse the existing throwing functions, but since the line numbers are hardcoded into the test, the test would break whenever the functions in `checkin:error` are moved around.